Run Information
========================
cognizant.intelligent.test.scripter.engine : ${project.version}
java.runtime.name    : OpenJDK Runtime Environment
java.version         : 11.0.10
java.home            : /usr/lib/jvm/java-11-amazon-corretto.x86_64
os.name              : Linux
os.arch              : amd64
os.version           : 4.14.225-169.362.amzn2.x86_64
file.encoding        : UTF-8
========================
Run Started on Thu Apr 01 04:48:11 PDT 2021
Loading Browser Profile
Loading RunManager
[Scenario:DMS_WorkFlow] [TestCase: Uat_SingleAttachment] [Description: Test Run] [Browser: Chrome] [BrowserVersion: default] [Platform: ANY][ExistingBrowser: false]
Initializing Report
ApiLink not available!!
Run Manager true
Continue Execution true
Shutting Down Thread as there is no need for SelectiveThreading
Inside Task Run
Running Iteration 1
Launching Chrome
Launching Local Driver
Starting ChromeDriver 89.0.4389.23 (61b08ee2c50024bab004e48d2b1b083cdbdac579-refs/branch-heads/4389@{#294}) on port 27780
Only local connections are allowed.
Please see https://chromedriver.chromium.org/security-considerations for suggestions on keeping ChromeDriver safe.
ChromeDriver was started successfully.
2021-04-01 04:48:11:04S -0700 [INFO] org.openqa.selenium.remote.ProtocolHandshake createSession:Detected dialect: W3C
Chrome Launched
Step:1   | Object: Browser | Action: AddVar | Input: Mail_Ingestion:Mail_UserID | Conditon: %OutlookUsername% | @01-Apr-2021 04:48:11.011
[DEBUG]   | Test Data Not Found, Iteration - 1 is missing. 
[Env : Default | Sheet : Mail_Ingestion | Field : Mail_UserID | TestCase : DMS_WorkFlow/Uat_SingleAttachment ]

===================================================================================================

2021-04-01 04:48:11:04S -0700 [SEVERE] com.cognizant.cognizantits.engine.core.Task onError:Error in testcase [//DMS_WorkFlow/Uat_SingleAttachment]
com.cognizant.cognizantits.engine.execution.exception.TestFailedException: Error in testcase [//DMS_WorkFlow/Uat_SingleAttachment]
	at com.cognizant.cognizantits.engine.execution.run.TestCaseRunner.onDataNotFoundException(TestCaseRunner.java:306)
	at com.cognizant.cognizantits.engine.execution.run.TestCaseRunner.run(TestCaseRunner.java:350)
	at com.cognizant.cognizantits.engine.core.Task.runIteration(Task.java:130)
	at com.cognizant.cognizantits.engine.core.Task.run(Task.java:85)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: Test Data Not Found, Iteration - 1 is missing. 
[Env : Default | Sheet : Mail_Ingestion | Field : Mail_UserID | TestCase : DMS_WorkFlow/Uat_SingleAttachment ]
	at com.cognizant.cognizantits.engine.execution.data.DataAccessInternal.throwErrorWithCause(DataAccessInternal.java:287)
	at com.cognizant.cognizantits.engine.execution.data.DataAccess.getData(DataAccess.java:71)
	at com.cognizant.cognizantits.engine.execution.data.DataProcessor.resolve(DataProcessor.java:64)
	at com.cognizant.cognizantits.engine.core.CommandControl.sync(CommandControl.java:172)
	at com.cognizant.cognizantits.engine.execution.run.TestStepRunner.executeStep(TestStepRunner.java:160)
	at com.cognizant.cognizantits.engine.execution.run.TestStepRunner.executeStep(TestStepRunner.java:147)
	at com.cognizant.cognizantits.engine.execution.run.TestStepRunner.run(TestStepRunner.java:66)
	at com.cognizant.cognizantits.engine.execution.run.TestCaseRunner.runStep(TestCaseRunner.java:395)
	at com.cognizant.cognizantits.engine.execution.run.TestCaseRunner.run(TestCaseRunner.java:346)
	... 5 more

[DEBUG]   | Error in testcase [//DMS_WorkFlow/Uat_SingleAttachment]

===================================================================================================


---------------------------------------------------
Testcase Name        : DMS_WorkFlow:Uat_SingleAttachment
Executed Steps       : 2
Passed Steps         : 0
Failed Steps         : 2
Time Taken           : 00:00:00
-----------------------------------------------------

Run Finished on Thu Apr 01 04:48:11 PDT 2021
-----------------------------------------------------
ExecutionDate        : 01-Apr-2021 04-48-11
Executed TestCases   : 1
Passed TestCases     : 0
Failed TestCases     : 1
Time Taken           : 00:00:00
ReportPath           : file:////home/ab773031t/Documents/cognizant-intelligent-test-scripter-1.4/Projects/PSIC_Digital_DMS/Results/TestDesign/DMS_WorkFlow/Uat_SingleAttachment/01-Apr-2021 04-48-11/summary.html
-----------------------------------------------------
